src/js/popup.js

```diff
@@ -194,6 +194,48 @@ function init() {
     chrome.i18n.getMessage("version", chrome.runtime.getManifest().version)
   );
 
+  $('#expand-blocked-resources').on('click', showBlockedResourcesHandler);
+  $('#collapse-blocked-resources').on('click', hideBlockedResourcesHandler);
+
+  $('#expand-firstparty-popup').on('click', showFirstPartyInfoHandler);
+  $('#collapse-firstparty-popup').on('click', hideFirstPartyInfoHandler);
+
+  if (POPUP_DATA.showExpandedTrackingSection) {
+    $('#expand-blocked-resources').hide();
+    $('#collapse-blocked-resources').show();
+    $('#blockedResources').show();
+  } else if (!POPUP_DATA.showExpandedTrackingSection) {
+    $('#expand-blocked-resources').show();
+    $('#collapse-blocked-resources').hide();
+    $('#blockedResources').hide();
+  }
+
+  $('#instructions-firstparty-description').hide();
+  $('#collapse-firstparty-popup').hide();
+
+  // check if any firstparty scripts are run on current tab & show message in popup
+  async function fetchFirstPartiesManifest() {
```

Comment:
Let's do the work of checking if we are on a first party protections page back in the background page. The popup should just get a new boolean as part of the popup data object.